rhizome/postgres/bin/configure

```diff
@@ -48,25 +48,27 @@ pg_hba_entries = <<-PG_HBA
 # Database administrative login by Unix domain socket
 local   all             postgres                                peer map=system2postgres
 local   all             pgbouncer                               peer map=system2pgbouncer
+local   replication     all                                     peer
+
+# Allow connections from localhost with ubi_monitoring OS user as
+# ubi_monitoring database user. This will be used by postgres_exporter
+# to scrape metrics and expose them to prometheus.
+local   all             ubi_monitoring                          peer
 
 # "local" is for Unix domain socket connections only
-local   all             all                                     peer
+# Use SCRAM authentication for all local connections to allow pgbouncer
+# passthrough to work for all non-unix users
+local   all             all                                     scram-sha-256
```

Comment:
It would be good to add a rule that allows peer authentication just for postgres user. Otherwise we would lose the ability to do psql directly while we are using postgres user.

Reply:
Would the 1st rule in the file work for that?

Reply:
Ah, yes it would. All is good then. :+1: